diff --git a/node_modules/react-native-payments/android/src/main/java/com/reactnativepayments/ReactNativePaymentsModule.java b/node_modules/react-native-payments/android/src/main/java/com/reactnativepayments/ReactNativePaymentsModule.java
index a47b50d..f26dd58 100644
--- a/node_modules/react-native-payments/android/src/main/java/com/reactnativepayments/ReactNativePaymentsModule.java
+++ b/node_modules/react-native-payments/android/src/main/java/com/reactnativepayments/ReactNativePaymentsModule.java
@@ -5,11 +5,11 @@ import android.view.WindowManager;
 import android.app.Activity;
 import android.content.Intent;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
-import android.support.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.NonNull;
 import android.app.Fragment;
 import android.app.FragmentManager;
-import android.support.annotation.RequiresPermission;
+import androidx.annotation.RequiresPermission;
 import android.util.Log;
 
 import com.facebook.react.bridge.Callback;
